### PR TITLE
Add GH Actions workflow to build React app

### PR DESCRIPTION
**Pull request by Subzeero**
* Configure a simple version of a workflow to build the React app with npm on node 16/18.

### CODE REVIEW

**Comment by Subzeero** [comment]
Since the React app isn't located at the root directory, I'll have to make a quick fix so it builds correctly.

**Comment by Subzeero** [comment]
There we go. The basic build of the app completes successfully.

File: my-webxr-app/src/App.tsx
Comment (matthew-buglass) [inline my-webxr-app/src/App.tsx:1]: Why are these files being changed?

File: my-webxr-app/src/Components/Button.tsx
Comment (matthew-buglass) [inline my-webxr-app/src/Components/Button.tsx:1]: Why are these files being changed?

File: my-webxr-app/src/Components/RotatingBox.tsx
Comment (matthew-buglass) [inline my-webxr-app/src/Components/RotatingBox.tsx:1]: Why are these files being changed?

**Review by matthew-buglass — CHANGES_REQUESTED** [review]

**Comment by Subzeero** [comment]
@matthew-buglass These files had unused imports that were preventing a successful build of the React app. These unused imports were simply removed. Reference: https://github.com/UniversityOfSaskatchewanCMPT371/term-project-2024-team-2/actions/runs/7674920319/job/20920370799

**Review by matthew-buglass — APPROVED** [review]

**Pull request merged**